core/app/modules/kit-library/data/kits-controller.php

```diff
@@ -12,7 +12,7 @@ class Kits_Controller extends Controller_Base {
 	/**
 	 * @var Repository
 	 */
-	private $repository;
+	public $repository;
```

Comment:
```suggestion
	private $repository;
```

use `getRepository()` method, then you can remove `//** @var Repository $repository */` then. if u need help with PHPDOC im here.